packages/web/utils/formatter.ts

```diff
@@ -392,3 +393,9 @@ export function calcFontSize(numChars: number, isMobile: boolean): string {
 
   return "16px";
 }
+
+export function formatSpendLimit(spendLimit: PricePretty | undefined) {
```

Comment:
Nit: thoughts on a more generic name for readability? A name that describes what it's doing more than what data it's displaying: like trimmedPrice or something

Reply:
f8273fb95988

thanks,